honestcash-v2/src/app/main/components/story-comment-editor/story-comment-editor.component.html

```diff
@@ -0,0 +1,33 @@
+<div *ngIf="user" class="card">
+  <div id="story-comment-wrapper" style="padding: 1.25rem;">
+    <div class="row">
+      <div class="col-1 pr-0">
+        <core-avatar [src]="user && user.imageUrl" large="true"></core-avatar>
+      </div>
+      <div class="col">
+        <p
```

Comment:
this code is duplicated in editor for the title aswell (expanding textarea). this will be refactored into a component in core that is reused